spotless/eclipse-java-formatter.xml

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="16">
+    <profile kind="CodeFormatterProfile" name="Spotless" version="16">
```

Comment:
Can we enforce that `public static final` (maybe all `static final`? not sure) vars are in ALL_CAPS_WITH_UNDERSCORES format?